query-node/mappings/src/common.ts

```diff
@@ -220,7 +220,7 @@ export function convertBytesToString(b: Bytes | null): string {
   const result = Buffer.from(b.toU8a(true)).toString()
 
   // prevent utf-8 null character
-  if (result.match(/^\0$/)) {
+  if (result.match(/^\0+$/)) {
```

Comment:
This is still a problem for strings that contain other characters.
For example I tried sending `createType('Bytes', 'abcabcabc\0\0\0\0\0')` as `handle` arg for `buyMembership` and it still crashed the query-node.

I would really recommend the well-tested `replace(/\u0000/g, '')` solution I used for olympia (https://github.com/Joystream/joystream/blob/olympia_dev/query-node/mappings/common.ts#L242)

Reply:
Thx for double-checking. `replace(/\u0000/g, '')` has been set there - fixed.